pkg/fftm/config_docs_generate_test.go

```diff
@@ -29,12 +29,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const configDocHeader = `---
```

Comment:
I note you don't actually need this. It was inherited from `firefly-core` where the docs were being generated into Jekyll format, and looks ugly when you view in Github.

The new interface allowing `GenerateConfigMarkdown` to accept a config header, means code (like here) can customize that header of it wants to.

Reply:
... where the simplest form of customization is just not to have a header at all

Reply:
You can see the ugly version here: https://github.com/hyperledger/firefly-transaction-manager/blob/main/config.md

Reply:
I went like-for-like - happy to change it to simply:

### Configuration reference

in both `config_docs_test.go` and `config_docs_generate_test.go` if GitHub is the only format we're going to use this in.

Reply:
Yeah - this only goes to Github